lib/contributionreward.js

```diff
@@ -159,17 +159,11 @@ class ContributionReward extends ExtendTruffleContract(
     // console.log(`********* options.externalTokenReward ${options.externalTokenReward} **********`);
```

Comment:
please remove commented code .

Reply:
@orenyodfat I hesitate to remove all commented code.  It's still in there because it can be useful.  I'm curious what is your motivation for removing it all as a rule?

Reply:
Code without commented code is much cleaner and readable.
If it is useful code so uncomment that.
If it useful for debug purposes so add it to a suitable debug log or something which can be controlled by the one who use this library (passing debug flag or something similar).
In general I think that code ,especially on a public repo should be clean ,readable and include what is needed for the code to operate. (comment are not).

Reply:
OK ok  ;-)

Reply:
@dkent600 seems like there are still some comments in this file to remove